### DISCUSSION TITLE
Is there a way to skip checking deps for a repository when renovate runs?

### DISCUSSION THREAD

**Discussion by fracai** [renovatebot/renovate#26302] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.93.1

### If you're self-hosting Renovate, select which platform you are using.

Gitea or Forgejo

### What is your question?

I recently added a repository with a large number of dependencies and the time to check deps went from around 1 minute, to over 9, even without opening PRs. This also happens to be a low priority project and I don't need to look at dependency updates more than weekly, maybe less. Is there a way to configure this specific project to not bother checking for updates outside of a schedule?

My current `renovate.json` is as follows, but this only restricts opening PRs and I'd like to skip checking entirely.
````
{
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
    "extends": [
        "schedule:earlyMondays",
        "config:recommended",
        "config:best-practices",
        "group:allNonMajor",
        ":separateMultipleMajorReleases"
    ]
}
````

Worst case I can reduce the schedule in general, but ideally I'd just skip this one repo.

Thanks for any tips.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
I think what you're asking for here is a "hard" schedule (do not clone, do not do anything) which is not currently supported however would be an acceptable feature if someone were to contribute it. It could potentially be a new boolean option `scheduleStrict` which means that if the repo itself has a `schedule` defined and it is not within schedule then the repo is fully skipped.

**Reply by fracai** [reply]
Yes, I think that would handle what I'm looking for.
For now I might create a second renovate branch that excludes the heavy repo.
Thanks
